scss/_patterns_accordion.scss

```diff
@@ -38,12 +38,12 @@
 
       // aria-selected controls the open and closed state for the accordion tab
       &[aria-expanded='true'] {
-        @include vf-icon-minus(vf-url-friendly-color($color-mid-dark));
+        @include vf-icon-minus($color-mid-dark);
```

Comment:
If this pattern is using mixins from icons shouldn't we import icons at the top so people can include just this pattern?